packages/passport/sdk-sample-app/src/components/zkevm/EthSendTransactionExamples/SpendingCapApproval.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { Accordion, Form } from 'react-bootstrap';
+import { usePassportProvider } from '@/context/PassportProvider';
+import WorkflowButton from '@/components/WorkflowButton';
+import { RequestExampleProps } from '@/types';
+import { Interface } from 'ethers/lib/utils';
+import { useImmutableProvider } from '@/context/ImmutableProvider';
+import { Environment } from '@imtbl/config';
```

Comment:
```suggestion
import { EnvironmentNames } from '@/types';
```
We should be using `EnvironmentNames` here, so that we also have `dev`